src/ontology/mondo-edit.obo

```diff
@@ -560881,7 +560883,7 @@ subset: gard_rare {source="MONDO:GARD"}
 subset: rare
 xref: OMIM:137215 {source="MONDO:includedEntryInOMIM"}
 is_a: MONDO:0020573 {source="OMIM:137215"} ! inherited disease susceptibility
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/1748 {source="OMIM:137215"} ! CDH1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/1748 ! CDH1
```

Comment:
This should have been protected https://github.com/monarch-initiative/omim/blob/main/data/protected-disease-gene.tsv#L24. I will add this back.